src/d3/lineFeature.js

```diff
@@ -105,6 +106,10 @@ var d3_lineFeature = function (arg) {
 
       m_renderer._drawFeatures(m_style);
     });
+    for (i = data.length; i < m_maxIdx; i += 1) {
```

Comment:
It was always required; not having it was a bug.  In the past, if you reduced the number of lines you had in the d3 renderer, they wouldn't be taken away.